### DISCUSSION TITLE
Cannot run CARLA in Docker

### DISCUSSION THREAD

**Discussion by ghost** [carla-simulator/carla#7003] [Q&A]
Hello everyone,

I want to run the CARLA Simulator in Docker. I pulled the latest CARLA image from Docker Hub and tried to run it. However, no matter which version of CARLA I am using, I always get the same error:

sh: 1: xdg-user-dir: not found

I followed the instructions in the CARLA documentation but still get the same error. I've been looking for a solution for a few hours now but couldn't find anything. I am using Windows 11, and I only have Docker and CARLA installed. Is it even possible to run everything from Windows? Because I only find tutorials and explanations for Linux.

I hope anyone can help me with my issue.

**Comment by Dev-WSun** [comment]
Same issue with ubuntu as host system.

**Reply by anshgwash** [reply]
Were you able to resolve this?

**Comment by Fillipko** [comment]
Running this for for CARLA 0.9.15 works on Ubuntu 22.04:
`docker run --privileged --gpus all --net=host -e DISPLAY=$DISPLAY --user $UID carlasim/carla:0.9.15 /bin/bash ./CarlaUE4.sh`

Not exactly sure what the root cause of the error is, but specifying the user ID helped. Annoying that the command shown in the documentation doesn't work.

**Reply by Cam-00** [reply]
It also works for me by specifying the user ID  for CARLA 0.9.13 working on Ubuntu 22.04.
THANKS a lot!
